postgres/src/main/java/org/jdbi/v3/postgres/PGObjectArrayArgumentFactory.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jdbi.v3.postgres;
+
+import java.sql.Array;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.sql.Types;
+
+import org.jdbi.v3.core.argument.AbstractArgumentFactory;
+import org.jdbi.v3.core.argument.Argument;
+import org.jdbi.v3.core.config.ConfigRegistry;
+import org.jdbi.v3.core.statement.StatementContext;
+import org.postgresql.util.PGobject;
+
+/**
+ * Argument factory for array of {@link PGobject}.
+ */
+public class PGObjectArrayArgumentFactory extends AbstractArgumentFactory<PGobject[]> {
+
+    public PGObjectArrayArgumentFactory() {
+        super(Types.ARRAY);
+    }
+
+    @Override
+    protected Argument build(PGobject[] value, ConfigRegistry config) {
+        return new Argument() {
```

Comment:
idem `LoggableBinderArgument`

Reply:
@TheRealMarnes Would you care to elaborate a little bit this?
I do not know how `LoggableBinderArgument` works, but it seems to me that this just bind any argument as a String.

I couldn't find documentation about this. I found some usage in code (ie. `JavaTimeZoneIdArgumentFactory` ).

Keep in mind that I want to bind an Array here.

Reply:
@TheRealMarnes Using LoggableBinderArgument now. I couldn't find any doc on this, so a double check on this would be greatly appreciated. Thanks